main.tex

```diff
@@ -18,6 +18,33 @@
 \begin{document}
 
 \maketitle
+\subsubsection {Facebook}
+Eine Studie von \textcite{empowermentThroughFacebook}, welche zwischen 2011 und 2012 in New Delhi durchgeführt wurde, zeigt, dass Facebook zu der Zeit eine große Rolle in der Gesellschaftsentwicklung spielte. 
+\medskip
+Da zu 2012 eine Regulierung von bis zu 200 SMS pro Tag veröffentlicht wurde(Press Information Bureau,
+Government of India, 2012), stieg das Interesse von Facebook zu der Zeit stark an,
+da Handybesitzer weiterhin unlimitiert mit ihren Freunden kommunizieren wollten.
+Durch das Kennenlernen und Nutzen von Facebook bildete sich eine Gesellschaft, welche bislang der Globalisierung
+durch das Internet ausgeschlossen waren und nun erstmals eine transnationale Identität erstellen konnte.
+\medskip
+Bis 2015 wuchs die Nutzeranzahl von Facebook in Indien geradezu explosionsartig, und mit derzeit 135 Millionen Nutzer waren gerademal 25\% weiblich\autocite{slideshareIndia}. Das liegt vorallem daran, dass vorallem jungen indischen Frauen, aus elterlichen Prinzipien, 
+wenig Freiheit gelassen wurde sich mit Freunden zu treffen, oder gar Technologie
+zu besitzen. Dieser Anteil sollte sich laut Aktivisten in der Zukunft stark ändern, 
+aber zu 2018 sind sogar 78\% der indischen Facebook Nutzer männlich. 
```

Comment:
Vorschlag:
Das liegt vor allem daran, dass junge indischen Frauen, aus elterlichen Prinzipien, weniger Anrecht auf  Technologien hatten. 
Dieser weibliche Anteil sollte sich, laut Aktivisten, in der Zukunft stark erhöhen, aber zu 2018 sind sogar 78\% der indischen Facebook Nutzer männlich.